docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -102,7 +102,7 @@ $ sudo usermod -a -G kvm,libvirt <username>
 . Update the *libvirt* configuration in *_/etc/libvirt/qemu.conf_* as follows:
 +
 ----
-$ sudo sed -r 's/group=".+"/group="kvm"/1' /etc/libvirt/qemu.conf > /etc/libvirt/qemu.conf
+$ sudo sed -ri 's/.group = ".+"/group = "kvm"/1' /etc/libvirt/qemu.conf
```

Comment:
Already mentioned in IRC, but I'm wary of the fragility of this `sed` expression. In particular, `.group` will match any single character before 'group', but not zero characters -- likewise, the white space is fragile if it changes again in the future.

:+1: for the in-place modification!